onboarding.md

```diff
@@ -34,6 +34,8 @@ onboarding session.
 * git:
   * Make sure you have whitespace=fix: `git config --global --add
     apply.whitespace fix`
+  * It is recommended to sign all commits under the Node.js repository.
+    Run: `git config commit.gpgsign true` inside the `node` folder.
```

Comment:
We should consider making this recommendation on the `ONBOARDING.md` of nodejs/Release, and maybe reword it to ensure we're not setting a bar too high for newcomers:
```suggestion
  * Consider configuring git to sign all commits when working on the Node.js repository.
    Run: `git config commit.gpgsign true` inside the `node` folder.
```

Reply:
I fear that whatever the wording, people are going to run the command, and later they will be confused as their `git commit` fail because of the missing signing key.

Reply:
Can it start with "if you have a GPG key setup locally that match the email you use to contribute to Node.js" so that people who do not know what it is will skip it?

Reply:
FWIW there are methods of signing commits with git. GitHub supports (i.e. can displayed "Verified") [GPG, SSH, or S/MIME signatures](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#about-commit-signature-verification).

For the release team we've recommend GPG keys, but this documentation is aimed at a broader audience (collaborators).